feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts

```diff
@@ -0,0 +1,312 @@
+// import { HttpClient } from "@angular/common/http";
+import {
+  HttpTestingController,
+  HttpClientTestingModule,
+} from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import {
+  QuoteList,
+  QUOTE_LIST_NORMALIZER,
+  Quote,
+  QUOTE_NORMALIZER,
+  QUOTE_STARTER_SERIALIZER,
+  Comment,
+  QUOTE_METADATA_SERIALIZER,
+  QUOTE_ACTION_SERIALIZER,
+  QuoteStarter,
+  QuoteMetadata,
+  QuoteAction,
+  QUOTE_COMMENT_SERIALIZER,
+  QuoteDiscount,
+  QUOTE_DISCOUNT_SERIALIZER,
+} from '@spartacus/commerce-quotes/core';
+import { ConverterService, OccConfig, OccEndpoints } from '@spartacus/core';
+import { take } from 'rxjs/operators';
+import { OccCommerceQuotesAdapter } from './occ-commerce-quotes.adapter';
+
+const userId = '111111';
+const cartId = '222222';
+const mockQuote: Quote = {
+  allowedActions: ['EDIT'],
+  cartId: cartId,
+  code: '333333',
+};
+
+const MockOccModuleConfig: OccConfig = {
+  backend: {
+    occ: {
+      baseUrl: '',
+      prefix: '',
+      endpoints: {
+        getQuotes: '/users/${userId}/quotes',
+        createQuote: '/users/${userId}/quotes',
+        getQuote: '/users/${userId}/quotes/${quoteCode}',
+        editQuote: '/users/${userId}/quotes/${quoteCode}',
+        performActionQuote: '/users/${userId}/quotes/${quoteCode}/action',
+        addComment: '/users/${userId}/quotes/${quoteCode}/comments',
+        addDiscount: '/users/${userId}/quotes/${quoteCode}/discounts',
+        addCartEntryComment:
+          '/users/${userId}/quotes/${quoteCode}/entries/${entryNumber}/comments',
+      } as OccEndpoints,
+    },
+  },
+};
+
+describe(`OccCheckoutDeliveryModesAdapter`, () => {
+  let service: OccCommerceQuotesAdapter;
+  // let httpClient: HttpClient;
```

Comment:
`should:` remove comment

Reply:
already removed with [ef44817](https://github.com/SAP/spartacus/pull/15742/commits/ef44817571204b0b95783311a669079033338612) commit